multiplatform-movies/common/build.gradle

```diff
@@ -1,11 +1,41 @@
 apply plugin: 'kotlin-multiplatform'
 apply plugin: 'kotlinx-serialization'
+apply plugin: 'com.codingfeline.buildkonfig'
+apply plugin: 'com.novoda.build-properties'
 
 // TODO work around for https://youtrack.jetbrains.com/issue/KT-27170
 configurations {
     compileClasspath
 }
 
+buildProperties {
+    android {
+        using project.file('android-secrets.properties')
+    }
+    ios {
+        using project.file('ios-secrets.properties')
+    }
+}
+
+buildkonfig {
```

Comment:
And [BuildKonfig](https://github.com/yshrsmz/BuildKonfig) to generate a `BuildKonffig` per platform.